microsoft-edge/extensions-chromium/index.md

```diff
@@ -102,6 +102,15 @@ To submit your extension to the store, you need to provide the following items:
 
 After you've successfully submitted your extension, your extension undergoes a review process and either passes or fails the certification process.  Owners are notified of the outcome and given next steps as required.  If you submit an extension update to the store, it begins a new review process.
 
+### The Featured badge in Edge Add-ons Store
+
+A **Featured extension** badge in the Microsoft Edge Add-ons Store indicates that the extension has been determined to follow Microsoft's recommended best practices related to security, privacy, transparency, and user experience. The Microsoft Edge Add-ons Store team reviews each extension for adherence to EAS best practices before granting it a **Featured** badge. 
+
```

Comment:
```suggestion

![The Featured badge for an extension](./index-images/featured-badge.png)
```

Add screenshot of the badge.

@Bhuvana-Priya - What does the tooltip say when you hover on a Featured badge?  What does badge look like?  It would be good to show a capture to actually show what a Featured badge looks like.

Reply:
![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/115227785/227512038-b8b5db0f-ad54-4ca2-916e-0c344bdd4599.png)

I attached a screenshot of how badge will look and this would be the Tooltip text: "This extension follows all of Microsoft's recommended practices and has been verified"

Reply:
fixed locally in my PR